Assets/Scripts/Player/PlayerController.cs

```diff
@@ -10,7 +10,7 @@ public class PlayerController : Entity {
 	const float hardLandDistance = .64f * 4f;
 	const float dashSpeed = 6f;
 	const float terminalFallSpeed = -10f;
-	const float dashCooldownLength = .6f;
+	public static float dashCooldownLength = .6f;
```

Comment:
Does this need to be public now? Looks like it's just being referenced in the PlayerController still.

Reply:
Yeah that was from a previous idea of how to implement the slide buff.